src/legacy-styles/components/_editor-override.scss

```diff
@@ -29,11 +29,10 @@
 	.editor,
```

Comment:
Style changes here are somewhat unrelated to the PR, though warranted.

Reply:
I'm so sorry. The CSS stuff is something I playing around with to fix other, more visual, stuff. It should not be part of this PR.